src/integrationtests/resources/subscriptionsForArtifactTest.json

```diff
@@ -0,0 +1,29 @@
+[
```

Comment:
Preferable it should be a subscriptionTemplate, then inject the data in the same template from different tests.

Reply:
I see your point but then i could just skip using the file. The reason i'm using the file is that i don't want this information cluttered in the test.

Reply:
All you need to change is 3-4 parameters in the file, not all 14. Thus the file is still needed. after few more tests will be a clutter of files.

Reply:
Ye that's true. I'll try that out and see how it looks :)